addons/website/controllers/main.py

```diff
@@ -202,10 +203,10 @@ def reset_template(self, templates, redirect='/'):
         return request.redirect(redirect)
 
     @http.route('/website/customize_template_get', type='json', auth='user', website=True)
-    def customize_template_get(self, xml_id, full=False, bundles=False):
-        """ Lists the templates customizing ``xml_id``. By default, only
-        returns optional templates (which can be toggled on and off), if
-        ``full=True`` returns all templates customizing ``xml_id``
+    def customize_template_get(self, key, full=False, bundles=False, **kw):
+        """ Get inherit view's informations of the template ``xml_id``. By default, only
```

Comment:
Ok. Shouldn't we add this explanation in the 'key' field ? help='"Full xml_id of the view, including the module that declares it: module.xml_id." I am sure this will help some people :p .